mission-control/docs/config-db/concepts/masking.md

```diff
@@ -0,0 +1,29 @@
+# Masking
+
+Masking allows replacing sensitive fields with hash of that field or with a static string.
+The field to mask is specified by the `jsonPath` config and `value` field defines the hash function or the static value.
+[See example](../examples/masking-fields).
+
+| Field      | Description                                                                                                         | Scheme                          | Required |
+| ---------- | ------------------------------------------------------------------------------------------------------------------- | ------------------------------- | -------- |
+| `selector` | Selector helps in choosing which configs should use the mask.                                                       | [`MaskSelector`](#maskselector) | `true`   |
+| `jsonpath` | Specify JSONPath expression for the fields                                                                          | `string`                        | `true`   |
+| `value`    | Value can be a name of a hash function or just a string. See [supported hash functions](#supported-hash-functions). | `string`                        | `true`   |
```

Comment:
Supported hash funcs should in scheme

.e.g. `string` or [`hashFunction'](./abc)